be/src/vec/core/block.cpp

```diff
@@ -518,7 +518,8 @@ std::string Block::dump_data(size_t begin, size_t row_limit, bool allow_null_mis
             if (data[i].column) {
                 if (data[i].type->is_nullable() && !data[i].column->is_nullable()) {
                     assert(allow_null_mismatch);
-                    s = assert_cast<const DataTypeNullable*>(data[i].type.get())
```

Comment:
not need disable,this is a debug function

Reply:
done